spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/ArgumentSanitizer.java

```diff
@@ -95,10 +99,12 @@ public String sanitize(String argument) {
 	 * @return the argument with a potentially sanitized value
 	 */
 	public String sanitize(String key, String value) {
-		for (Pattern pattern : this.keysToSanitize) {
-			if (pattern.matcher(key).matches()) {
-				value = REDACTION_STRING;
-				break;
+		if (value != null && StringUtils.hasText(value)) {
```

Comment:
`value ! = null` can be removed as `StringUtils.hasText(value)` does the non null check as well.